external/os-autoinst-common/.github/workflows/base-commit-message-checker.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Check subject line length
         uses: gsactions/commit-message-checker@v2
         with:
-          pattern: '^.{1,72}(\n|$)'
+          pattern: '^(Revert "?)?.{1,72}(\n|$)'
```

Comment:
Does this count `Revert`+72chars right? which exceeds the limit of 72 IIUC! And it makes the error msg incorrect. Just making sure that this is what we want.

Reply:
This is an update of the external repo. If it's wrong, then it needs to be fixed upstream, so please file an issue there